tests/logs/problem2_Mon 25 Jul 2022 05:01:24 PM EDT.log

```diff
@@ -0,0 +1,1385 @@
+============================= test session starts ==============================
```

Comment:
All of these log files shouldn't be included

Reply:
I have removed then